packages/react/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -875,7 +877,7 @@ MultiSelect.propTypes = {
    *     locale: string,
    *   }) => Array<Item>
    */
-  sortItems: PropTypes.func.isRequired,
+  sortItems: PropTypes.func,
```

Comment:
This is an optional prop to override the `defaultSortItems` behavior. Should not be required.